Thanks for this collab!

Can you try to fix the Travis built? It cannot find cpplocate.

`CMake Error at source/llassetgen-rendering/CMakeLists.txt:8 (find_package):
  By not providing "Findcpplocate.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "cpplocate", but CMake did not find one.
  Could not find a package configuration file provided by "cpplocate" with
  any of the following names:
    cpplocateConfig.cmake
    cpplocate-config.cmake
  Add the installation prefix of "cpplocate" to CMAKE_PREFIX_PATH or set
  "cpplocate_DIR" to a directory containing one of the above files.  If
  "cpplocate" provides a separate development package or SDK, be sure it has
  been installed.`

--- a/.travis.yml
+++ b/.travis.yml
@@ -51,7 +51,7 @@ before_install:
     if [ $TRAVIS_OS_NAME == osx ]; then
       brew tap cginternals/homebrew-cginternals;
       brew update && brew install cppcheck fontconfig freetype qt5 glm glbinding cginternals/homebrew-cginternals/globjects cginternals/homebrew-cginternals/cpplocate;
-      export CMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH;$(brew --prefix qt);$(brew --prefix glbinding);$(brew --prefix globjects);$(brew --prefix cpplocate)";
+      export CMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH;$(brew --prefix qt);$(brew --prefix glm);$(brew --prefix glbinding);$(brew --prefix globjects);$(brew --prefix cpplocate)";
     fi
   - chmod +x ./configure
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -45,13 +45,13 @@ before_install:
       sudo apt-add-repository ppa:cginternals/ppa -y;
       sudo apt-add-repository ppa:cginternals/backports-ppa -y;
       sudo apt-get update -qq;
-      sudo apt-get install -qq qt5-default libglm-dev libglbinding-dev libglobjects-dev;
+      sudo apt-get install -qq qt5-default libglm-dev libglbinding-dev libglobjects-dev libcpplocate-dev;
     fi  # libpng is already installed by default
   - |-
     if [ $TRAVIS_OS_NAME == osx ]; then
       brew tap cginternals/homebrew-cginternals;
-      brew update && brew install cppcheck fontconfig freetype qt5 glm glbinding cginternals/homebrew-cginternals/globjects;
-      export CMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH;$(brew --prefix qt);$(brew --prefix glbinding);$(brew --prefix globjects)";
+      brew update && brew install cppcheck fontconfig freetype qt5 glm glbinding cginternals/homebrew-cginternals/globjects cginternals/homebrew-cginternals/cpplocate;
+      export CMAKE_PREFIX_PATH="$CMAKE_PREFIX_PATH;$(brew --prefix qt);$(brew --prefix glbinding);$(brew --prefix globjects);$(brew --prefix cpplocate)";
     fi
   - chmod +x ./configure
 

